tests/test_ufonormalizer.py

```diff
@@ -794,64 +792,18 @@ def test_normalizeGLIF_note_defined(self):
         _normalizeGlifNote(element, writer)
         self.assertEqual(
             writer.getText(),
-            "<note>\n\tA quick brown fox jumps over the lazy dog.\n\t"
+            "<note>A quick brown fox jumps over the lazy dog.\n"
             "P\u0159\xedli\u0161 \u017elu\u0165ou\u010dk\xfd k\u016f\u0148 "
-            "\xfap\u011bl \u010f\xe1belsk\xe9 \xf3dy.\n</note>")
-
-        element = ET.fromstring(
-            "<note>   Line1  \t\n\n    Line3\t  </note>")
-        writer = XMLWriter(declaration=None)
-        _normalizeGlifNote(element, writer)
-        self.assertEqual(
-            writer.getText(),
-            "<note>\n\tLine1\n\t\n\t    Line3\n</note>")
-
-        # Normalizer should not indent Line2 and Line3 more than already indented
-        element = ET.fromstring(
-            "<note>\n\tLine1\n\tLine2\n\tLine3\n</note>")
-        writer = XMLWriter(declaration=None)
-        _normalizeGlifNote(element, writer)
-        self.assertEqual(
-            writer.getText(),
-            "<note>\n\tLine1\n\tLine2\n\tLine3\n</note>")
-
-        # Normalizer should keep the extra tab in line 2
-        element = ET.fromstring(
-            "<note>\n\tLine1\n\t\tLine2\n\tLine3\n</note>")
-        writer = XMLWriter(declaration=None)
-        _normalizeGlifNote(element, writer)
-        self.assertEqual(
-            writer.getText(),
-            "<note>\n\tLine1\n\t\tLine2\n\tLine3\n</note>")
-
-        # Normalizer should keep the extra spaces on line 2
-        element = ET.fromstring(
-            "<note>\n\tLine1\n\t    Line2\n\tLine3\n</note>")
-        writer = XMLWriter(declaration=None)
-        _normalizeGlifNote(element, writer)
-        self.assertEqual(
-            writer.getText(),
-            "<note>\n\tLine1\n\t    Line2\n\tLine3\n</note>")
-
-        # Normalizer should remove the extra tab all lines have in common,
-        # but leave the additional tab on line 2
-        element = ET.fromstring(
-            "<note>\n\t\tLine1\n\t\t\tLine2\n\t\tLine3\n</note>")
-        writer = XMLWriter(declaration=None)
-        _normalizeGlifNote(element, writer)
-        self.assertEqual(
-            writer.getText(),
-            "<note>\n\tLine1\n\t\tLine2\n\tLine3\n</note>")
+            "\xfap\u011bl \u010f\xe1belsk\xe9 \xf3dy.</note>")
 
-        # Normalizer should remove the extra 4-space all lines have in common,
-        # but leave the additional 4-space on line 2
+        # Everything is always preserved
         element = ET.fromstring(
-            "<note>\n        Line1\n            Line2\n        Line3\n</note>")
+            "<note>\n\tLine1\n\t\tLine2\n\t    Line3\n</note>")
         writer = XMLWriter(declaration=None)
         _normalizeGlifNote(element, writer)
         self.assertEqual(
             writer.getText(),
-            "<note>\n\tLine1\n\t    Line2\n\tLine3\n</note>")
+            "<note>\n\tLine1\n\t\tLine2\n\t    Line3\n</note>")
 
```

Comment:
Can you please add a test for an XML entity (`&`, `<`, or `>`) to make sure that is handled correctly?